cypress/pageobject/Users/ManageUserPage.ts

```
@@ -164,6 +172,14 @@ export class ManageUserPage {
         });
       });
   }
+
+  interceptFacilitySearchReq() {
+    cy.intercept("GET", "**/api/v1/facility/**").as("searchFacility");
+  }
```

Biome

[error] Duplicate class member name "interceptFacilitySearchReq"



(lint/suspicious/noDuplicateClassMembers)